scripts/joern/setup.sh

```diff
@@ -3,6 +3,7 @@
 mkdir external
```

Comment:
```suggestion
mkdir -p external
```
Let's pass -p to handle the situation when directory already exists.